tests/dash/test_dash_privatelink.py

```diff
@@ -24,6 +24,39 @@
 """
 
 
+def get_dpu_dataplane_port(duthost, dpu_index):
+    platform = duthost.facts["platform"]
+    platform_json = json.loads(duthost.shell(f"cat /usr/share/sonic/device/{platform}/platform.json")["stdout"])
+    try:
+        interface = list(platform_json["DPUS"][f"dpu{dpu_index}"]["interface"].keys())[0]
+    except KeyError:
+        interface = f"Ethernet-BP{dpu_index}"
```

Comment:
In the latest cisco platform code, Ethernet-BP naming convention is no longer used.